src/app/examples/card/card-elevations-example/card-elevations-example.component.html

```diff
@@ -0,0 +1,20 @@
+<kirby-card elevation="disabled">
+  <kirby-card-header title="Disabled" subtitle='elevation="disabled"'></kirby-card-header>  
+  <div class="card-content">
+    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Mollitia facere molestias recusandae necessitatibus.
+  </div>
+</kirby-card>
+
+<kirby-card elevation="default">
```

Comment:
Remove `default` as it's already... well... the default 👅 
```suggestion
<kirby-card>
```